First/src/firstPackage/EqualOccurances.java

```diff
@@ -0,0 +1,28 @@
+package firstPackage.First.src.firstPackage;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class EqualOccurances {
+    public static boolean numberOfOccurences(String input) {
+        Map<Character, Integer> countOccurances = new HashMap<>();
+        char[] charOne = input.toCharArray();
+        for (char ch : charOne) {
```

Comment:
variable names should be according to points shared by Amit

Reply:
DONE !